src/test/java/net/bramp/ffmpeg/FFprobeTest.java

```diff
@@ -57,6 +58,13 @@ public void testVersion() throws Exception {
   }
 
   @Test
+  public void testStartPtsType()
```

Comment:
Sure, seems like a much more complete way than my simple 'make sure no one has changed this from a long'.